.github/workflows/ci.yml

```diff
@@ -134,13 +134,13 @@ jobs:
       - name: Run installer.sh with preview version
         env:
           VERBOSE: ${{ runner.debug }}
-          VERSION: 0.0.0-preview.v76e03eb68
+          VERSION: preview
```

Comment:
this does not take any effect since you're overriding this in line 139, we should have a test with `preview` as a version as well